utils/rpms/daos.spec

```
@@ -360,6 +361,7 @@ mkdir -p %{buildroot}/%{_sysconfdir}/ld.so.conf.d/
 echo "%{_libdir}/daos_srv" > %{buildroot}/%{_sysconfdir}/ld.so.conf.d/daos.conf
 mkdir -p %{buildroot}/%{_sysctldir}
 install -m 644 utils/rpms/%{sysctl_script_name} %{buildroot}/%{_sysctldir}
+#install -m 644 utils/rpms/%{limits_script_name} %{buildroot}/%{_limitsdir}
```

@brianjmurrell could you advise how to properly specify the _limitsdir ?

---

I'm not aware of a pre-defined `%{_limitsdir}` on either of EL8 or Leap 15.3:

```
$ rpm --eval %{_limitsdir}
%{_limitsdir}
```

```suggestion
install -m 644 utils/rpms/%{limits_script_name} %{buildroot}/%{_sysconfdir}/security/limits.d/
```